.reuse/dep5

```diff
@@ -7,5 +7,6 @@ Files: flake.lock
        README.md
        **/README.md
        pkgs/surfer/Cargo.lock
+       pkgs/xmake/macosx.patch
```

Comment:
I think we should just match all patch files, given that we now have 2 of them.

Reply:
I considered this but wasn't sure if it was best to be explicit.

Should I do the same for all `*.lock` files as well while I'm at it?